### PR TITLE
🗃 Start admonitions in the open position

### LINKED ISSUE #1319: allow dropdown admonitions to start open
currently an admonition declared with `:class: dropdown` always starts closed
there are cases where it makes sense to have both the ability to open/close *and* to start open

## Proposal

add an `:open: ` flag to the `admonition` directive; its default would then be `false` if the `dropdown` class is mentioned, and `false` otherwise

### PR DESCRIPTION
**Pull request by rowanc1**
Fixes #1319

Requires: https://github.com/jupyter-book/myst-theme/pull/452

### CODE REVIEW

File: docs/admonitions.md [hunk 1]
```diff
@@ -167,6 +167,7 @@ This is the body.
 To turn an admonition into a dropdown, add the `dropdown` {myst:directive}`admonition.class` to them.
 Dropdown admonitions use the `<details>` HTML element (meaning they also will work without Javascript!),
 and they can be helpful when including text that shouldn't immediately visible to your readers.
+To have the admonition be a dropdown, and also start open, add the {myst:directive}`admonition.open` option.
```
Comment (agoose77) [inline docs/admonitions.md:170]:
```suggestion
To have a dropdown-style admonition start open, add the {myst:directive}`admonition.open` option.
```

Comment (agoose77) [reply docs/admonitions.md:170]: The `open` option doesn't add the `class` for dropdown, right?

Comment (agoose77) [reply docs/admonitions.md:170]: Perhaps we should add the `dropdown` class if `open` is defined?

Comment (agoose77) [reply docs/admonitions.md]: @rowanc1 should we warn the user if they don't define `class: dropdown` then?

Comment (rowanc1) [reply docs/admonitions.md]: Yeah - I will add that. Right now it just doesn't do anything silently.

**Comment by rowanc1** [comment]
- [ ] Add error if there is a clash between dropdown and open options

**Review by fwkoch — COMMENTED** [review]
This is good - not changing the default but adding a simple `open` option.

Rather than introducing validation errors if `dropdown` and `open` disagree, could we just have `open: true/false` imply `class: dropdown`?

**Comment by rowanc1** [comment]
We took @fwkoch's suggestion of coercing and working either way!

**Pull request merged**